target/linux/ath79/image/generic.mk

```diff
@@ -128,6 +128,15 @@ define Device/buffalo_wzr-hp-g450h
 endef
 TARGET_DEVICES += buffalo_wzr-hp-g450h
 
+define Device/comfast_cf-e110n
+  ATH_SOC := qca9533
+  DEVICE_TITLE := COMFAST CF-E110N
+  DEVICE_PACKAGES := rssileds kmod-leds-gpio -swconfig -uboot-envtools
+  IMAGE_SIZE := 16192k
+  SUPPORTED_DEVICES += cf-e110n
```

Comment:
You can drop `SUPPORTED_DEVICES` as `SUPPORTED_DEVICES := comfast,cf-e110n` it's added automatically in `Device/Default` in target/linux/ath79/image/Makefile.

Reply:
Done.